packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -222,6 +224,9 @@ const start = async (zcf, privateArgs) => {
 
   const { brand: runBrand } = await E(runMint).getIssuerRecord();
 
+  let mintedRUN = AmountMath.makeEmpty(runBrand);
+  let burnedRUN = AmountMath.makeEmpty(runBrand);
```

Comment:
just noting the variable names should continue to match the metric names

Reply:
done